### DISCUSSION TITLE
Set Keyboard shortcut to navigate to a "Shortcut"

### DISCUSSION THREAD

**Discussion by v1s10n-4** [dailydotdev/daily#971] [Feature Requests]
After setting a "Shortcut", it is only reachable using a mouse. The only way to trigger it via keyboard is using the tab key navigation which is painful. It could be awesome to set a real keyboard shortcut to trigger the "Shortcut".
![Screenshot 2023-09-06 at 13 37 00](https://github.com/dailydotdev/daily/assets/16701409/0daad479-a6cb-4129-966c-f7c4fc906ca9)

**Comment by idoshamun** [comment]
Hi @v1s10n-4,

Thanks for reporting this accessibility issue! 🙏
